Common/AR/source/LocationDataSource.cpp

```diff
@@ -148,29 +213,20 @@ QGeoPositionInfoSource* LocationDataSource::geoPositionSource() const
 
 /*!
   \brief Sets the position source to \a geoPositionSource.
+
+  The geoposition source can not be changed when the tracking is started.
```

Comment:
Question: If `geoposition` is a single word, then shouldn't the property be named `geopositionSource`?

Reply:
The Qt's type is `QGeoPositionInfoSource`, I don't know what is the best word to use.